### PR TITLE
Make it work

### PR DESCRIPTION
**Pull request by swebb**

### CODE REVIEW

File: lib/rails_easy_sign_in.rb [hunk 1]
```diff
@@ -1,2 +1,6 @@
-class RailsEasySignIn
+module RailsEasySignIn
+  VERSION = "0.0.4"
```
Comment (jordanmaguire) [inline lib/rails_easy_sign_in.rb]: If it works just make it `1.0.0`

File: README.md [hunk 2]
```diff
@@ -1 +1,17 @@
-# Rails Easy Sign In
\ No newline at end of file
+# Rails Easy Sign In
+
+Adds a sign in box to your login page to allow automatic sign in. Useful in development environments.
+
+## Installation
+
+  1. Add  ```rb gem 'chosen-rails' ``` to the development group of your _Gemfile_ and bundle.
```
Comment (jordanmaguire) [inline README.md]: What's the deal with this line?

Comment (jordanmaguire) [inline README.md]: Oh right - presuming you copied from somewhere else.

File: app/assets/javascripts/jordan.js [hunk 3]
```diff
@@ -1 +0,0 @@
-alert("yolo")
```
Comment (jordanmaguire) [inline app/assets/javascripts/jordan.js]: classic jordan

File: lib/rails_easy_sign_in.rb (hunk 1 above)
Comment (swebb) [inline lib/rails_easy_sign_in.rb]: There's probably some things we could still improve. I wouldn't call it ready for 1.0.0 yet.

**Comment by jordanmaguire** [comment]
let's fuck this donkey

**Pull request merged**